products/early_access_features/backend/api.py

```diff
@@ -60,6 +63,10 @@ class Meta:
     def update(self, instance: EarlyAccessFeature, validated_data: Any) -> EarlyAccessFeature:
         stage = validated_data.get("stage", None)
 
+        request = self.context["request"]
+        user_data = UserBasicSerializer(request.user).data if request.user else None
+        serialized_previous = MinimalEarlyAccessFeatureSerializer(instance).data
```

Comment:
@daibhin This is breaking tests because this crashes when the `EarlyAccessFeature` instance doesn't include an attached `feature_flag`. Working on a fix.

Reply:
Apologies, don't know how that passed tests. Thanks for fixing